### PR TITLE
Migrate back to Discord.py V2

### PR DESCRIPTION
**Pull request by D0rs4n**
- Migrate to use the newly announced Async Cog loading feature
- That's it, also the message content intent has been enabled.

### CODE REVIEW

File: bot/__init__.py [hunk 1]
```diff
@@ -14,6 +13,3 @@
 
 
 setup_logging()
-
-# Apply all monkey patches from bot core.
-apply_monkey_patches()
```
Comment (ChrisLovering) [inline bot/__init__.py:19]: These are still needed

File: bot/bot.py [hunk 2]
```diff
@@ -97,4 +105,17 @@ async def wait_until_guild_available(self) -> None:
         await self._guild_available.wait()
 
 
-bot = SirRobin(command_prefix=constants.Client.prefix, activity=disnake.Game("The Not-Quite-So-Bot-as-Sir-Lancebot"))
+_intents = discord.Intents.default()  # Default is all intents except for privileged ones (Members, Presences, ...)
```
Comment (ChrisLovering) [inline bot/bot.py:108]: We likely want `_intents.members` too, but isn't needed by any cogs right now.

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

**Review by ChrisLovering — APPROVED** [review]

File: bot/bot.py [hunk 3]
```diff
@@ -97,4 +105,18 @@ async def wait_until_guild_available(self) -> None:
...
```
Comment (Bluenix2) [inline bot/bot.py:108]: Seeing how many we're disabling. Would it be easier using `discord.Intents.none()`?

Comment (D0rs4n) [reply bot/bot.py:108]: I mean that's how it's done on Lance, it's a bit more easier to understand this way, the result is basically the same. :D

**Review by ToxicKidz — APPROVED** [review]

**Pull request merged**